workers/unity/Assets/Gdk/Physics/Systems/ApplyTransformUpdatesSystem.cs

```diff
@@ -21,8 +23,24 @@ protected override void OnUpdate()
         {
             for (var i = 0; i < transformUpdateData.Length; i++)
             {
-                var newUpdates = transformUpdateData.TransformUpdate[i].Buffer;
-                transformUpdateData.BufferedTransform[i].TransformUpdates.AddRange(newUpdates);
+                var transformUpdates = transformUpdateData.TransformUpdate[i].Buffer;
+                var lastTransformSnapshot = transformUpdateData.BufferedTransform[i].LastTransformSnapshot;
+                var transformSnapshots = new List<SpatialOSTransform>();
+                foreach (var update in transformUpdates)
+                {
+                    lastTransformSnapshot.Location = update.Location.HasValue
```

Comment:
It may look better to do an if block here

Reply:
Fixed.